editor/import/post_import_plugin_skeleton_renamer.cpp

```diff
@@ -130,8 +131,10 @@ void PostImportPluginSkeletonRenamer::_internal_process(InternalImportCategory p
 
 		TypedArray<Node> nodes = p_base_scene->find_children("*");
```

Comment:
How about adding the `BoneAttachment3D` filter here to avoid fetching all nodes unnecessarily? Basically merging in https://github.com/godotengine/godot/pull/83149.

Reply:
> Okay, I've modified it now to remove the bind, and use casting instead. I didn't put the change @dpalais made in case you want to merge that seperately, though I can put it in mine too if you'd prefer this all be one PR. It's not strictly nessecary, but it results in less looping and feels like good practice.

All in one commit would be better IMO. You could attribute @dpalais as co-author with their info from https://patch-diff.githubusercontent.com/raw/godotengine/godot/pull/83149.patch